test/sanitize-field-names/koa.test.js

```diff
@@ -5,6 +5,13 @@
  */
 
 'use strict';
+
+const isKoaIncompat = require('../_is_koa_incompat')();
+if (isKoaIncompat) {
+  console.log(`# SKIP: ${isKoaIncompat}`);
+  process.exit(0);
+}
```

Comment:
I think (from the test failure) you need to add this block to test/instrumentation/modules/koa/set-framework.test.js as well.